Firmware/config.h

```diff
@@ -3,7 +3,15 @@
 #define HW_VERSION 1
 
 // the temperature the system is supposed to not exceed
-#define TEMP_MAX 400
+#define TEMP_MAX 450
```

Comment:
Yes, there should be two maximum temperatures: one that the PID controller is allowed to use as target temperature and one that is the absolute maximum that the system can measure while every higher temperature than this results in a "tip error",